src/main/resources/hudson/tasks/junit/pipeline/JUnitWrapperPipelineStep.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.tasks.junit.pipeline
+
+import org.jenkinsci.plugins.workflow.cps.CpsScript
+
+class JUnitWrapperPipelineStep implements Serializable {
+    CpsScript script
+
+    JUnitWrapperPipelineStep(CpsScript script) {
+        this.script = script
+    }
+
+    def call(Map args, Closure closure) {
```

Comment:
Uh, what?? Just make a plain old `Step`.

In fact this DSL is just sugar over the existing `SimpleBuildStep` and so is less valuable than the original PR, which

* allowed you to inspect results in Groovy code and make decisions based on them
* allowed you to decide whether or not to set the build status